french/script/story/st_00.rpy

```diff
@@ -6,201 +6,201 @@ translate french s00_last_time_on_jpde_265b952a:
     voice "audio/vo/narrator/006.ogg"
-    na ""
+    na "À travers les actions et la résistance, enhardis par les liens uniques entre individus de toutes origines et de tous horizons.{w=0.3} Se trouvant eux-mêmes involontairement au centre de tout cela et pourtant, n'ayant jamais abandonné le combat."
 
 # game/script/story/st_00.rpy:46
 translate french s00_last_time_on_jpde_347597fc:
 
     # voice "audio/vo/narrator/007.ogg"
     # na "Three years have passed since that fateful day.{w=0.3} Remnant, and her people, have been forever changed..."
     voice "audio/vo/narrator/007.ogg"
-    na ""
+    na "Trois ans ont passés depuis ce jour fatidique.{w=0.3} Remnant, et son peuple, ont été changé à jamais..."
 
 # game/script/story/st_00.rpy:51
 translate french s00_last_time_on_jpde_43550bbf:
 
     # voice "audio/vo/narrator/008.ogg"
     # na "And now darkness encroaches again."
     voice "audio/vo/narrator/008.ogg"
-    na ""
+    na "Et maintenant les ténèbres gagnent à nouveau du terrain."
 
 # game/script/story/st_00.rpy:61
 translate french s00_birth_of_the_red_fang_3ef06803:
 
     # voice "audio/vo/narrator/009.ogg"
     # na "[dis_wday], [dis_day] [dis_month] 492.{w=0.3} Vacuo, border region with the Kingdom of Vale.{w=0.5} An ambitious man, set on a path of violence and retribution, arrives at an ancient fortress with an armed entourage."
     voice "audio/vo/narrator/009.ogg"
-    na ""
+    na "[dis_way], [dis_day] [dis_month] 492.{w=0.3} Vacuo, région frontalière avec le Royaume de Vale.{w=0.5} Un homme ambitieux, engagé sur le chemin de la violence et du châtiment, arrive dans une ancienne forteresse avec un entourage armé."
 
 # game/script/story/st_00.rpy:69
 translate french s00_birth_of_the_red_fang_002406c9:
 
     # voice "audio/vo/narrator/010.ogg"
     # na "He has come to remove an obstacle, one way or the other.{w=0.3} What hangs in the balance..."
     voice "audio/vo/narrator/010.ogg"
-    na ""
+    na "Il est venu se débarasser d'un obstacle, d'une manière ou d'une autre.{w=0.3} Ce qui est en jeu..."
 
 # game/script/story/st_00.rpy:71
 translate french s00_birth_of_the_red_fang_d8b3c072:
 
     # sk_x darko neutral "I guess congratulations are in order, {b}Adam{/b}.{w=0.5} You've done it."
-    sk_x darko neutral ""
+    sk_x darko neutral "Je suppose que les félicitations sont de mise, {b}Adam{/b}.{0.5} Tu y es parvenu."
 
 # game/script/story/st_00.rpy:76
 translate french s00_birth_of_the_red_fang_2046b873:
 
     # adt nulla ".{w=0.3}.{w=0.3}."
-    adt nulla ""
+    adt nulla ".{w=0.3}.{w=0.3}."
 
 # game/script/story/st_00.rpy:77
 translate french s00_birth_of_the_red_fang_44b20e6b:
 
     # voice "audio/vo/narrator/011.ogg"
     # na "Is the leadership of the {b}White Fang{/b}, and the future of the fauni people."
     voice "audio/vo/narrator/011.ogg"
-    na ""
+    na "La dirigeante du {b}White Fang{/b}, et le futur du peuple faunus."
```

Comment:
c'est la suite de "Ce qui est en jeu...", j'avais pas vu.

Pas besoin de "Is" ça fait moche en français "est nanana".

La dirigeance(c'est un peu moche mais ça existe), le commandement?

Reply:
```
"Ce qui est en jeu..."

[...]

"... Est le commandement du [...]"
```

Tu en penses quoi ?

Reply:
Oui, en FR, tu peux couper la phrase, c'est une question en suspens "Ce qui est en jeu? Le commandement [...]"

Reply:
Ok j'applique ma p'tite solution du coup !